unblob/extractor.py

```diff
@@ -46,6 +47,45 @@ def fix_permissions(outdir: Path):
             path.chmod(0o664)
 
 
+def is_safe_path(basedir: Path, path: Path) -> bool:
```

Comment:
Just a fun thing to keep in mind (this has already came up in `logger.py`):

In python 3.9, there is an `is_relative_to` function in `pathlib`:
https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.is_relative_to